### ISSUE TITLE
Simple echoing ValueLatch<T> node.

### ISSUE THREAD

**Issue by whatsavalue3** [Yellow-Dog-Man/Resonite-Issues#3433]
### Is your feature request related to a problem? Please describe.

I don't like having to create a new local variable for a constant return value.

### Describe the solution you'd like

`ValueLatch<T>` and `ObjectLatch<T>`
### Inputs:
```
Call <ISyncOperation>
Input <T>
```
### Outputs:
```
Next <Continuation>
Value/Object <T>
```
Will return the Input on the Value/Object output when called. Will do nothing else. Its just to cache the Input in the Value/Object output. It will benefit performance and make caching values much faster.

Optionally, a `ValueMultiLatch<T>` and `ObjectMultiLatch<T>` can be added that has a variable amount of impulses. Letting you set the same variable from different parts of the code, while also continuing to different places.

### Describe alternatives you've considered

Using a ValueWrite<T> on a Local. That is clunky and less performant.

### Requesters

`@WhatsAValue3`

**Comment by Frooxius** [comment]
We already have the Write Latch nodes. Do these note serve this purpose? 

What kind of scenarios do you want to use these in?

**Comment by whatsavalue3** [comment]
![Image](https://github.com/user-attachments/assets/cdd04b32-7da8-40da-ab49-6922964caa38)
Instead of using a variable, i could use the node itself as the value output.

**Comment by whatsavalue3** [comment]
I have to do a lot of these, having a single node that will do it efficiently will be a game-changer

**Comment by whatsavalue3** [comment]
Do you have any more questions? @Frooxius

**Comment by Krzeszny** [comment]
I hope you don't mind mentions but it seems like you missed a reply @Frooxius 

If I understand correctly, it's an optimized Write that would allow for more performant contraptions?

I've been following Wav's projects and they're often limited by the speed of node clusters. To show support, I've made some mock-ups. I hope it meets your needs for a quick and useful QoL feature:

![Image](https://github.com/user-attachments/assets/92db74c4-e875-4158-ba27-5a6269f56988)

**Comment by JackTheFoxOtter** [comment]
Direct writes are already very optimized. You can accomplish the same by just writing to a local variable.

**Comment by JackTheFoxOtter** [comment]
In fact I'm pretty sure if this were implemented, it would literally just be putting a write and a local variable into the same node. So this could also be accomplished once we have #564 by creating a custom node for this operation.

**Comment by AmasterAmaster** [comment]
I guess the coding paradigm is just different (from other different platforms) and could be clashing here, but data storage can be arranged using slots too, especially for modularity. At least that is what I do when I need to store something to be used later (similar to a latch).

Although I like the idea of a "ready-to-use" "already-set" variable from it for the convenience, that is what writes do and probably best to have initialization somewhere else in the code that handled the "latching"? (if that makes sense)

**Comment by whatsavalue3** [comment]
> I hope you don't mind mentions but it seems like you missed a reply [@Frooxius](https://github.com/Frooxius)
> 
> If I understand correctly, it's an optimized Write that would allow for more performant contraptions?
> 
> I've been following Wav's projects and they're often limited by the speed of node clusters. To show support, I've made some mock-ups. I hope it meets your needs for a quick and useful QoL feature:
> 
> ![Image](https://github.com/user-attachments/assets/92db74c4-e875-4158-ba27-5a6269f56988)

Yes, something like that, but have it just return the given value when impulsed. Then you can just reuse the outputted value like its a constant.
It would speed up coding quite a bit.

**Comment by AmasterAmaster** [comment]
@whatsavalue3 Depending on how crafty you are, you can technically make a custom node (similar to the above image) that contains a dynamic var containing your value, then just make an output for what you need (no inputs required :3).

**Comment by whatsavalue3** [comment]
@AmasterAmaster I dont want a dynamic variable. I want a local.
